cmake/arch.cmake

```diff
@@ -80,10 +79,15 @@ if (DYNAMIC_ARCH)
       string(REGEX REPLACE "-march=native" "" CMAKE_C_FLAGS "${CMAKE_C_FLAGS}")
     endif ()
     if (DYNAMIC_LIST)
-	set(DYNAMIC_CORE PRESCOTT ${DYNAMIC_LIST})
+      set(DYNAMIC_CORE PRESCOTT ${DYNAMIC_LIST})
     endif ()
   endif ()
 
+  CHECK_INCLUDE_FILE ("${PROJECT_SOURCE_DIR}/config_kernel.h" TRAP)
```

Comment:
You probably wanted `if(EXISTS ${PROJECT_SOURCE_DIR}/config_kernel.h)`?

Reply:
CHECK_INCLUDE_FILE worked, but your suggestion certainly looks like the more elegant solution, thanks.